RealmSwift.podspec

```diff
@@ -0,0 +1,31 @@
+Pod::Spec.new do |s|
+  s.name                    = 'RealmSwift'
+  s.version                 = `sh build.sh get-version`
+  s.summary                 = 'Realm is a modern data framework & database for iOS & OSX.'
+  s.description             = <<-DESC
+                              Realm is a modern data framework & database for iOS & OSX. You can use it purely in memory — or persist to disk with extraordinary performance.
+
+                              Realm’s data structures look like NSObjects and NSArrays, but provide additional features such as: querying, relationships & graphs, thread safety, and more.
```

Comment:
How about these?

**Realm.podspec**

This pod is the official Objective-C API for Realm: a modern data framework &
database for iOS & OSX. You can use it purely in memory — or persist to disk
with extraordinary performance.

If you're looking for the Swift API, use `pod "RealmSwift"`.

Realm’s data structures look like regular objects and arrays, but provide
additional features such as: querying, relationships & graphs, thread safety,
encryption, multi-process sharing and more.

Realm is not built on SQLite. Instead, a custom C++ core is used to provide
memory-efficient access to your data by using Realm objects, which usually
consume less RAM than native objects. The core also provides an optional
persistence layer that can automatically save and retrieve your objects from
disk.

Realm offers extraordinary performance compared to SQLite and other persistence
solutions, and is currently powering apps on over 100 million devices.

**RealmSwift.podspec**

RealmSwift is the official Swift API for Realm: a modern data framework &
database for iOS & OSX. You can use it purely in memory — or persist to disk
with extraordinary performance.

If you're looking for the Objective-C API, use `pod "Realm"`.

Realm’s data structures look like regular objects and arrays, but provide
additional features such as: querying, relationships & graphs, thread safety,
encryption, multi-process sharing and more.

Realm is not built on SQLite. Instead, a custom C++ core is used to provide
memory-efficient access to your data by using Realm objects, which usually
consume less RAM than native objects. The core also provides an optional
persistence layer that can automatically save and retrieve your objects from
disk.

Realm offers extraordinary performance compared to SQLite and other persistence
solutions, and is currently powering apps on over 100 million devices.